src/mono/netcore/System.Private.CoreLib/src/System/Threading/ThreadPool.Browser.Mono.cs

```diff
@@ -76,36 +82,20 @@ internal static void RequestWorkerThread()
             QueueCallback();
         }
 
-        internal static bool KeepDispatching(int startTickCount)
-        {
-            return true;
-        }
-
         internal static void NotifyWorkItemProgress()
         {
         }
 
-        internal static bool NotifyWorkItemComplete()
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static bool NotifyWorkItemComplete(object? threadLocalCompletionCountObject, int currentTimeMs)
         {
             return true;
         }
 
-        private static RegisteredWaitHandle RegisterWaitForSingleObject(
-             WaitHandle waitObject,
-             WaitOrTimerCallback callBack,
-             object? state,
-             uint millisecondsTimeOutInterval,
-             bool executeOnlyOnce,
-             bool flowExecutionContext)
-        {
-            if (waitObject == null)
-                throw new ArgumentNullException(nameof(waitObject));
-
-            if (callBack == null)
-                throw new ArgumentNullException(nameof(callBack));
+        internal static object? GetOrCreateThreadLocalCompletionCountObject() => null;
 
+        private static void RegisterWaitForSingleObjectCore(WaitHandle? waitObject, RegisteredWaitHandle registeredWaitHandle) =>
```

Comment:
Which public API does call this?

Reply:
This is called by variants of `ThreadPool.RegisterWaitForSingleObject`. The null checks and some common code were moved to the shared `ThreadPool.cs` in the commit titled "Merge RegisteredWaitHandle implementations".